README.md

```diff
@@ -4,6 +4,12 @@
 
 Generate and push OpenTelemetry Trace data to an endpoint in JSON format.
 
+On local collector `http://localhost:4317 (gRPC only)`, run :
```

Comment:
You don't run that command on the collector. The collector runs (somewhere) and you run the `python` command on localhost. Notice we don't specify anywhere **where** the collector is. `--export-traces` **assumes** that the collector is running on localhost at the standard OTEL gRPC port `:4317`.

If you aren't running running the collector at `localhost:4317` you can set an environment variable to point pytest to a different collector endpoint. Explained here: https://github.com/chrisguidry/pytest-opentelemetry#installation-and-usage

Reply:
Ok, I try to change the sentence :) 
Please lemme know any idea to give the proper explanation for it :)